src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Colors.values().length);
```

Comment:
A new Random class instance is created each time the method is called. It's bad practice to initialize dependencies in the method. You can initialize it in fields, but good practice initialization using a constructor.